api/api.go

```diff
@@ -28,8 +28,8 @@ import (
 var (
 	xssMdlwr xss.XssMw
 
-	// AdminAddress is the eth address of the admin account
-	AdminAddress = "0x7E4A2359c745A982a54653128085eAC69E446DE1"
+	// AdminUser is the eth address of the admin account
+	AdminUser = "postables"
```

Comment:
this seems incredibly dangerous haha, especially with a public codebase, and we really shouldn't be locking admin functionality behind hardcoded users - at the very least, this should be an environment variable, and this should only be done in some kind of dev mode.

We should update how the users are managed in the database to include data on their permission level, ie admin, and use that instead